src/emailaction.h

```diff
@@ -457,7 +458,8 @@ class EasInvitationResponse : public EmailAction
     QMailProtocolAction* _protocolAction;
     QMailAccountId _accountId;
     int _response;
```

Comment:
It almost feels like `_response` could be an enum which might lessen some checks, but that's outside the scope here. Just crossed my mind.